### DISCUSSION TITLE
Compilation out of VS2022

### DISCUSSION THREAD

**Discussion by ghost** [CosmosOS/Cosmos#2137] [Q&A]
Hello guys, can I make Operating System in C# and Cosmos Without cosmos itself?
I have VS2022 and I don't want to install VS2019 or any other visual studio and on VS2022 the compilation isn't work.
If I can do that, How?
thx For answer, and sorry for my bad english

**Comment by quajak** [comment]
There is a branch you have to check out for vs2022 support currently.
